### ISSUE TITLE
list-style-type gets destroyed

### ISSUE THREAD

**Issue by haf** [jakubpawlowicz/clean-css#596]
On line 0:

```
  6 .list-123 {
  5   font-size: 1.2rem;
  4 
  3   padding: 0;
  2   margin-top: 2rem;
  1 
  0   list-style-type: none;
  1   list-style-type: decimal !ie; /*IE 7- hack*/
  2 
  3   counter-reset: li-counter;
  4 }
```

Gets output to:

`.list-123{font-size:1.2rem;padding:0;margin-top:2rem}`

**Comment by jakubpawlowicz** [comment]
clean-css runs in IE9+ mode by default, so first of all you need to run in IE7 compatibility mode (--compatibility ie7), however it still does not seem to work unless you turn off advanced mode (`--skip-advanced`). I'll find out why.

**Comment by jakubpawlowicz** [comment]
It's just been released with clean-css 3.4. :fireworks:

**Comment by haf** [comment]
:+1: thank you!
